index.d.ts

```diff
@@ -0,0 +1,20 @@
+declare module 'react-native-iot-wifi' {
+  // tslint:disable:no-namespace
+  type RemoveConnectArgs =
+    | [string, (error: string) => void]
+    | [string, boolean, (error: string) => void];
+
+  type ConnectSecureArgs =
+    | [string, (error: string) => void]
+    | [string, boolean, (error: string) => void];
+
+  export namespace RNWifi {
+    function isAvailable(cb: (available: boolean) => void): void;
+    function getSSID(cb: (ssid: string) => void): void;
+    function connect(...args: RemoveConnectArgs): void;
```

Comment:
Same `RemoveConnectArgs` used for connect method. Creating a separate type would be good for maintenance